tests/test_composition.py

```diff
@@ -97,3 +101,40 @@ def test_animationgroup_with_wait():
     timings = animation_group.anims_with_timings
 
     assert timings == [(wait, 0.0, 1.0), (sqr_anim, 1.0, 2.0)]
+
+
+@pytest.mark.parametrize(
+    "animation_remover, animation_group_remover", [(False, True), (True, False)]
+)
+def test_animationgroup_is_passing_remover_to_animations(
+    animation_remover, animation_group_remover
+):
+    scene = Scene()
```

Comment:
This makes me realize that there should be a `Scene` factory, so people won't have to instantiate a scene in a test.

Reply:
That's true, it would be also great to inject it to tests as fixture, as in graphic tests

Reply:
Yes, totally !